### ISSUE TITLE
TypeError: 'NoneType' object is not iterable

### ISSUE THREAD

**Issue by qingchunlizhi** [ColinDuquesnoy/QDarkStyleSheet#116]
/home/duan/anaconda3/envs/venv/lib/python2.7/site-packages/qdarkstyle/init.py:181: FutureWarning: load_stylesheet() will not receive pyside parameter in version 3. Set QtPy environment variable to specify the Qt binding insteady.
FutureWarning
Traceback (most recent call last):
File "/home/duan/pix2vox/ui/gui_main.py", line 115, in 
slider.valueChanged.connect(lambda value, j=int(index): self.opt_engine.set_label(float(value)/100, j))
File "/home/duan/pix2vox/opt/constrained_opt.py", line 59, in set_label
self.update_voxel_model()
File "/home/duan/pix2vox/opt/constrained_opt.py", line 42, in update_voxel_model
self.preprocess_constraints()
File "/home/duan/pix2vox/opt/constrained_opt.py", line 35, in preprocess_constraints
color_old, edge_old = self.constraints
TypeError: 'NoneType' object is not iterable

**Comment by dpizetta** [comment]
Hi @qingchunlizhi, this warning is normal to appear when executing debug or turn on warnings. 

```
/home/duan/anaconda3/envs/venv/lib/python2.7/site-packages/qdarkstyle/init.py:181: FutureWarning: load_stylesheet() will not receive pyside parameter in version 3. Set QtPy environment variable to specify the Qt binding insteady.
FutureWarning
```

The bottom error is probably in your application. Try it without using qdarkstyle. Your self.constraints is None, and it should be [None, None] if necessary because you need a return list color_old, edge_old.

Closing.
